src/components/Field/Field.css

```diff
@@ -28,20 +25,6 @@
   background: transparent;
 }
 
-.dcl.field.full .ui.input input,
```

Comment:
Would you mind explaining why were all these styles removed? Won't this break the current fields?